EasyPost/Models/API/CarrierFields.cs

```diff
@@ -47,12 +48,6 @@ public class CarrierField : EasyPostObject
     {
         #region JSON Properties
 
-        /// <summary>
-        ///     The key of the field.
-        /// </summary>
-        [JsonProperty("key")]
-        public string? Key { get; set; }
-
```

Comment:
Why is this removed?

Reply:
Because it does not exist in the response JSON schema. Instead, the "key" is the actual key in the key-value dictionaries.